src/main/jbake/content/hawkular-services/docs/quickstart-guide/index.adoc

```diff
@@ -55,20 +55,272 @@ endif::[]
 ----
 
 This would be the result of that command:
 
 .Command result
 ----
-[{"timestamp":1468533600000,"value":10.0},{"timestamp":1468535400000,"value":10.0},{"timestamp":1468537200000,"value":9.7},{"timestamp":1468539000000,"value":9.6},{"timestamp":1468540800000,"value":9.6},{"timestamp":1468542600000,"value":9.6},{"timestamp":1468544400000,"value":9.5},{"timestamp":1468546200000,"value":9.5},{"timestamp":1468548000000,"value":9.5},{"timestamp":1468549800000,"value":9.5},{"timestamp":1468551600000,"value":9.5},{"timestamp":1468553400000,"value":9.5},{"timestamp":1468555200000,"value":9.6},{"timestamp":1468557000000,"value":9.6},{"timestamp":1468558800000,"value":9.7},{"timestamp":1468560600000,"value":9.7},{"timestamp":1468562400000,"value":9.8},{"timestamp":1468564200000,"value":10.0},{"timestamp":1468566000000,"value":10.2},{"timestamp":1468567800000,"value":10.3},{"timestamp":1468569600000,"value":10.0},{"timestamp":1468571400000,"value":9.9},{"timestamp":1468573200000,"value":10.1},{"timestamp":1468575000000,"value":9.7},{"timestamp":1468576800000,"value":9.2},{"timestamp":1468578600000,"value":9.2},{"timestamp":1468580400000,"value":9.9},{"timestamp":1468582200000,"value":11.0},{"timestamp":1468584000000,"value":12.4},{"timestamp":1468585800000,"value":13.3},{"timestamp":1468587600000,"value":14.3},{"timestamp":1468589400000,"value":13.6},{"timestamp":1468591200000,"value":11.0},{"timestamp":1468593000000,"value":10.2},{"timestamp":1468594800000,"value":10.1},{"timestamp":1468596600000,"value":10.3},{"timestamp":1468598400000,"value":10.9},{"timestamp":1468600200000,"value":11.4},{"timestamp":1468602000000,"value":11.7},{"timestamp":1468603800000,"value":11.5},{"timestamp":1468605600000,"value":11.1},{"timestamp":1468607400000,"value":10.6},{"timestamp":1468609200000,"value":10.1},{"timestamp":1468611000000,"value":9.6},{"timestamp":1468612800000,"value":9.2},{"timestamp":1468614600000,"value":9.1},{"timestamp":1468616400000,"value":9.1},{"timestamp":1468618200000,"value":9.1}]
+[
+  {
+    "timestamp": 1468578600000,
+    "value": 9.2
+  },
+  {
+    "timestamp": 1468580400000,
+    "value": 9.9
+  },
+  {
+    "timestamp": 1468582200000,
+    "value": 11
+  },
+  {
+    "timestamp": 1468584000000,
+    "value": 12.4
+  },
+  {
+    "timestamp": 1468585800000,
+    "value": 13.3
+  },
+  {
+    "timestamp": 1468587600000,
+    "value": 14.3
+  },
+  {
+    "timestamp": 1468589400000,
+    "value": 13.6
+  },
+  {
+    "timestamp": 1468591200000,
+    "value": 11
+  },
+  {
+    "timestamp": 1468593000000,
+    "value": 10.2
+  },
+  {
+    "timestamp": 1468594800000,
+    "value": 10.1
+  }
+]
 ----
+
+It is likely that the data will need to be downsampled, if there are too many data points in a short period of time. For instance to display two years of weather
+in a graph, the precision every 30min is likely too much. The timeseries database allow to get "buckets" of data, which could be determined by the number of
```

Comment:
perhaps the word 'frequency' is more suitable here than 'precision', but it's a nit